test/integration/thirdparty/client-go_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package thirdparty
+
+import (
+	"testing"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubernetes/test/integration/framework"
+
+	exampletprv1 "k8s.io/client-go/examples/third-party-resources/apis/tpr/v1"
+	exampleclient "k8s.io/client-go/examples/third-party-resources/client"
+	examplecontroller "k8s.io/client-go/examples/third-party-resources/controller"
+	"context"
+)
+
+func TestClientGoThirdPartyResourceExample(t *testing.T) {
+	_, s := framework.RunAMaster(framework.NewIntegrationTestMasterConfig())
+	defer s.Close()
+
+	config := &rest.Config{Host: s.URL, ContentConfig: rest.ContentConfig{NegotiatedSerializer: api.Codecs}}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
```

Comment:
nit: I think at least some other packages use `assert.NoError(err)`/`require.NoError(err)` to reduce boilerplate.